tools/cmd-docs/main.go

```diff
@@ -37,6 +38,11 @@ func main() {
 		log.Fatal(err)
 	}
 
+	err = doc.GenMarkdownTree(argocddex.NewCommand(), "./docs/operator-manual/server-commands")
```

Comment:
Looks like lines 41-44 duplicate lines 46-49. Is it intentional?

Reply:
lines 41-44 are for argocd-dex whereas lines 46-49 are for argocd-util. 
Difference is between line 41 and 46. `argocddex.NewCommand()` vs `argocdutil.NewCommand()`

Reply:
oh, sorry. Did not notice dex in package name.